bigquery/google/cloud/bigquery/job.py

```diff
@@ -27,7 +27,9 @@
 from google.cloud.bigquery.dataset import Dataset
 from google.cloud.bigquery.dataset import DatasetListItem
 from google.cloud.bigquery.dataset import DatasetReference
+from google.cloud.bigquery.encryption_configuration import EncryptionConfiguration
```

Comment:
cleanup from another PR?

Reply:
Yeah, recently added for models encryption but I forgot to ask for the imports to be alphabetized.